docs/feature_split_keyboard.md

```diff
@@ -132,28 +132,23 @@ To enable this method, add the following to your `config.h` file:
 #define EE_HANDS
 ```
 
-Next, you will have to flash the EEPROM files once for the correct hand to the controller on each halve. You can do this manually with the following bootloader targets while flashing the firmware:
+Next, you will have to flash the EEPROM files once for the correct hand to the controller on each halve. You can do this manually with the following bootloader targets using `qmk flash -kb <keyboard> -km <keymap> -bl <bootloader>` command to flash:
```

Comment:
I was going to submit a PR to deal with some issues after this merged but looks like we might as well deal with this here.

> Next, you will have to flash the EEPROM files once

This doesn't apply to all controller and bootloader combinations.

Blackpill + tinyuf2, can be flashed once and it'll retain it's handedness in emulated eeprom when flashed again with a non-handed firmware.

Blackpill + dfu, has to be flashed every time with handed firmware as the emulated eeprom section is at the start of flash and overwritten.

I'm undecided if flash once should just be removed for simplicity or expanded.

```suggestion
Next, you will have to flash the correct handedness option to the controller on each halve. You can do this manually with the following bootloader targets using `qmk flash -kb <keyboard> -km <keymap> -bl <bootloader>` command to flash:
```
Maybe a note later?
```
?> Some controllers may not retain the handedness option when being flashed again, in this case you'll need to always flash each half with the correct bootloader target from above.
```
could probably be worded better.

Reply:
Added as an exception footnote to avoid wordiness in the previous paragraph.